src/chromatix/typing.py

```diff
@@ -0,0 +1,6 @@
+from jax import Array
+from numpy import ndarray
+
+ArrayLike = Array | ndarray
+NumberLike = ArrayLike | float | int
```

Comment:
Not important but I'm not sure I like this name, maybe `ScalarOrArray` or something? Is there any example in e.g. numpy or scipy we could look at?

Reply:
Also not sure if I like that we combine floats and ints together here but we can get to that later I guess.

Reply:
I changed it to ScalarLike.